ci_scripts/expect_scripts/SafeMath.exp

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/expect
+
+# uncomment line below for debugging
+# exp_internal 1
+
+set timeout 7
+
+spawn ./roc_nightly/roc run ./examples/SafeMath/main.roc
+
+
+expect "147.66666666666666\r\n" {
```

Comment:
The current implementation outputs `σ² = 147.66666666666666\r\n`

(tested with `roc run --linker=legacy` and `roc --version` == `roc nightly pre-release, built from commit ef05ec6 on Di 30 Mai 2023 09:34:00 UTC`)

Reply:
Good catch :)